pkg/config/sample-config.txt

```diff
@@ -0,0 +1,26 @@
+version = 31
+credential.accessKey = xxx
+credential.secretKey = yyy    ## SecretKey cannot be changed
+region = us-east-1
+browser = on
+worm = on
+cache.drives = 
+cache.expiry = 111
+cache.maxuse = 100
+cache.exclude = 
+notify.redis = on
+notify.redis.format = namespace
+notify.redis.address = 
+notify.redis.password = 
+notify.redis.key = 
+notify.webhook = on
+notify.webhook.endpoint = 
+log.http = on
+log.http.anonymous = on
+log.http.audit = on
+log.http.endpoint = https://username:password@example1.com/api
+log.subnet = off
```

Comment:
this sample can be incomplete or can be stored inside your test file itself as a verbatim string like 

```
var config string = `.... sample config... `
```

Reply:
@harshavardhana,
The reason I had it this way is to make my testing code mimic the exact scenario on the server side where we'll read/write from/to a config file.

Reply:
The affect is same, we will read it into a byte array so the processing wise it's the same. Unless you have a few samples of valid and invalid input I don't think we need this as a separate file.

Reply:
The exception is the 2 functions that are executed before processing the data in the server side testing logic:
 `readLines()` and  `writeLines()`.
If I change the file to a string, then these 2 functions won't be executed.

Reply:
What i meant is you can keep them as `bytes.Reader([]byte(string))`

Reply:
Here are the 2 functions:
```
// readLines reads a whole file into memory
// and returns a slice of its lines.
func readLines(path string) ([]string, error) {
	file, err := os.Open(path)
	if err != nil {
		return nil, err
	}
	defer file.Close()

	var lines []string
	scanner := bufio.NewScanner(file)
	for scanner.Scan() {
		lines = append(lines, scanner.Text())
	}
	return lines, scanner.Err()
}

// writeLines writes the lines to the given file.
func writeLines(lines []string, path string) error {
	file, err := os.Create(path)
	if err != nil {
		return err
	}
	defer file.Close()

	w := bufio.NewWriter(file)
	for _, line := range lines {
		fmt.Fprintln(w, line)
	}
	return w.Flush()
}
```
So, you basically want me to replace `bufio` package with `bytes` ?
Could you explain what the benefits would be other than including the sample file in api_test.go?